djstripe/models/core.py

```diff
@@ -762,6 +762,36 @@ def create(cls, subscriber, idempotency_key=None, stripe_account=None):
 
         return customer
 
+    @classmethod
+    def _get_or_create_from_stripe_object(
+        cls,
+        data,
+        field_name="id",
+        refetch=True,
+        current_ids=None,
+        pending_relations=None,
+        save=True,
+        stripe_account=None,
+    ):
+
+        from .billing import TaxId
+
+        res = super(Customer, cls)._get_or_create_from_stripe_object(
+            data,
+            field_name=field_name,
+            refetch=refetch,
+            current_ids=current_ids,
+            pending_relations=pending_relations,
+            save=save,
+            stripe_account=stripe_account,
+        )
+        if data.get("tax_ids"):
```

Comment:
could use a newline above the if block to separate the logic a bit.

Reply:
Also, this could be combined with the logic within the if. It might be less clear when reading though; up to you if you want to change it.

`tax_id_data = data.get("tax_ids", {}).get("data", [])`